extensions/src/ACRE2Core/FilterOcclusion.h

```diff
@@ -11,7 +11,11 @@ class CFilterOcclusion
 public:
     CFilterOcclusion(void);
     ~CFilterOcclusion(void);
-    ACRE_RESULT process(short* samples, int sampleCount, int channels, ACRE_VOLUME volume, Dsp::Filter *&filter);
+    ACRE_RESULT process(int16_t *const samples, const int32_t sampleCount, const int32_t channels, const ACRE_VOLUME volume, Dsp::Filter *&filter);
 
-    DECLARE_MEMBER(int, ChannelCount);
+    virtual __inline void setChannelCount(const int32_t value) { this->m_channelCount = value; }
```

Comment:
__inline is MSVC specific, `inline` is the real keyword.
Also inline doesn't make much sense on virtual functions. If the compiler doesn't know exactly that the class will be that exact type, then it won't be able to inline. And if it knew that it was that type, then it would also inline if you don't put the inline keyword there.